source/helpers/render-conventional-commit-types.ts

```diff
@@ -0,0 +1,31 @@
+import zipTextNodes from 'zip-text-nodes';
+
+// Using https://www.conventionalcommits.org/ as a reference.
+
+export function getConventionalCommitAndScopeMatch(commitTitle: string): RegExpExecArray | undefined {
+	const match = /^(?<type>\w+)(?:\((?<scope>.+)\))?: .+$/.exec(commitTitle);
+
+	if (!match) {
+		return;
+	}
+
+	return match;
+}
+
+export function removeCommitAndScope(
```

Comment:
This doesn’t remove the commit

```suggestion
export function cleanConventionalCommitElement(
```

Reply:
Sorry it should've said type and scope, not commit and scope

Reply:
Yep, that’s why I suggested this name (with either `clean` or `strip`)